MaxText/inference_mlperf/evaluate-accuracy-fast.py

```diff
@@ -42,13 +42,13 @@ def split_data(preds: List[str], refs: List[str], num_chunks: int) -> List[Tuple
 
 
 def compute_rouge_chunk(chunk: Tuple[List[str], List[str]], metric) -> Dict:
-  """Compute ROUGE scores for a chunk of data"""
```

Comment:
These fixes seem mistaken

Reply:
@suexu1025 should it be rouge instead of rogue :)

Reply:
![xmen-deal-with-it](https://github.com/user-attachments/assets/12c2d06d-2fd4-4c1d-a0f5-8b0046fa104b)